### DISCUSSION TITLE
Please add `groq`, `grok`, `cerebras` and `https://cerebrascoder.com` as API providers

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [cline/cline#977] [Feature Requests]
Please add `groq`, `grok`, `cerebras` and `https://cerebrascoder.com` as API providers

**Comment by HusainMkd** [comment]
cerebras coder is out of this world because of the speed!
please do add it, it gives a completely new approach to coding when you can instantly iterate

**Reply by timothygray** [reply]
Grok 3 is amazing and would love to use it inside Cline!

**Comment by mausch** [comment]
groq is OpenAI-compatible already, no need to support it explicitly https://console.groq.com/docs/openai

grok too https://docs.x.ai/docs/tutorial#step-4-make-a-request-from-python-or-javascript

**Comment by edwin-truthsearch-io** [comment]
Grok and Cerebras are supported, only Groq is missing. There is a PR for that, #3898. Please upvote!
